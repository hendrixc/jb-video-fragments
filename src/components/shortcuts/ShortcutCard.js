import React, { Component } from 'react';
import {
  Card,
  CardMedia,
} from '@rmwc/card';
import { Typography } from '@rmwc/typography';
import colors from '../../style/colors';
import I18n from '../../i18n';
import '@material/card/dist/mdc.card.css';
import '@material/button/dist/mdc.button.css';
import '@material/icon-button/dist/mdc.icon-button.css';
import '@material/typography/dist/mdc.typography.css';
import './shortcut.css';

class ShortcutCard extends Component {
  render() {
    return (
      <div>
        <Card>
          <CardMedia style={{ backgroundColor: colors.secondary500 }}>
            <div>
              <p className="shortcut">
                <Typography use="button" className="titleVideo">{I18n.t('clip.next')}</Typography>
              </p>
              <div className="shortcut">
                <Typography use="button" className="titleVideo">{I18n.t('clip.previous')}</Typography>
              </div>
            </div>
          </CardMedia>
        </Card>
      </div>
    );
  }
}

export default ShortcutCard;
