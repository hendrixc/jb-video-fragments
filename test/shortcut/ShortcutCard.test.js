import React from 'react';
import renderer from 'react-test-renderer';
import ShortcutCard from '../../src/components/shortcuts/ShortcutCard';

it('renders an card component into the ShortcutCard as snapshot', () => {
  const TextInputComponent = renderer.create(<ShortcutCard />).toJSON();
  expect(TextInputComponent).toMatchSnapshot();
});
