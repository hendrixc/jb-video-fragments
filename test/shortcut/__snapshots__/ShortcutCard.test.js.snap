// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders an card component into the ShortcutCard as snapshot 1`] = `
<div>
  <div
    className="mdc-card"
  >
    <section
      className="mdc-card__media"
      style={
        Object {
          "backgroundColor": "#E91E63",
        }
      }
    >
      <div>
        <p
          className="shortcut"
        >
          <span
            className="titleVideo mdc-typography--button"
          >
            Next: ctrl+shift+d
          </span>
        </p>
        <div
          className="shortcut"
        >
          <span
            className="titleVideo mdc-typography--button"
          >
            Previous: ctrl+shift+a
          </span>
        </div>
      </div>
    </section>
  </div>
</div>
`;
